### ISSUE TITLE
[Algo] ViTDet algo development

### ISSUE THREAD

**Issue by wenmengzhou** [alibaba/EasyCV#3]
reimplement the work described in https://arxiv.org/abs/2203.16527

**Comment by zugofn** [comment]
Hi!
Thank you for your great code job!
Do you have plan to implement the convolutional propagation in the paper?

**Comment by wenmengzhou** [comment]
> Hi! Thank you for your great code job! Do you have plan to implement the convolutional propagation in the paper?

yes,  we have been discussing the implementation details,  welcome to join in this discussion

**Comment by tuofeilunhifi** [comment]
> Hi! Thank you for your great code job! Do you have plan to implement the convolutional propagation in the paper?

The accuracy of conv aggreation we reproduced is 42.13, so we guess that pre-training of conv aggregation is required
